go-controller/pkg/factory/factory.go

```diff
@@ -535,6 +535,13 @@ func NewClusterManagerWatchFactory(ovnClientset *util.OVNClusterManagerClientset
 		}
 	}
 
+	if config.OVNKubernetesFeature.EnableInterconnect && config.OVNKubernetesFeature.EnableMultiNetwork {
+		wf.informers[PodType], err = newQueuedInformer(PodType, wf.iFactory.Core().V1().Pods().Informer(), wf.stopChan, defaultNumEventQueues)
```

Comment:
not sure if we need a queued informer with 15  workers here, the amount of work being done is very small, and you have a giant lock on the cache you use anyway.

Reply:
The (new) lock just affects releases, not allocations, and just to lookup in the cache.

The work is indeed small, but incurs in the IO of updating the pod annotation (with retries).

Since you are not sure, and now I am not sure either, let's keep it as is and have a discussion about it and make any changes on a separate PR